provisioning/playbook.yml

```diff
@@ -4,11 +4,11 @@
   become: yes
   become_method: sudo
   tasks:
-    - name: "Stop nginx"
-      service:
-        name: nginx
-        state: stopped
-      ignore_errors: yes
+  - name: "Stop nginx"
+    service:
+      name: "{{ user }}"
```

Comment:
This should be the name of the nginx service rather than the user, even if they are the same value. 

I'd also suggest it would be clearer to be a bit more explicit about the name of each variable, possibly `nginx_servicename`, `nginx_username`?

Or am I misunderstanding?

Reply:
Also, hello, and thanks!

Reply:
Yeah you are correct with all of those statements. I just made the assumption that the service and the user are the same- usually that's the case for nginx. Definitely not true for centos and debian for apache though... 

Yeah making the variables more explicit is definitely a good idea and improves readability. So no misunderstanding there at all. I was just consolidating the variables, and hey that's what collaborating is all about, right? 

So if the user and the service is the same it should actually be 

```
nginx:
  nginx_username: nginx
  nginx_servicename: nginx

service: "{{ nginx.nginx_servicename }}"
user: "{{ nginx.nginx_username }}"

```
That was just a contrived example for shortening the variable names. Which now that I think about it might actually fail as user might be a reserved value for ansible.